core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,235 @@
+                          <span class="mif-minus"></span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div id="infobox" class="info-box" data-role="infobox">
+                <span class="button square closer"></span>
+                <div class="info-box-content">
+                    <h3>{{ infoboxTitle }}</h3>
+                    <p>{{ infoboxMessage }}</p>
+                </div>
+            </div>
+
+        </div>
+    </div>
+</body>
+
+<script>
+    /*global Metro axios, Vue*/
+
+    new Vue({
+      el: '#app',
+      data: function () {
+        return {
+          newEndpoint: {
+            name: '',
+            owner: '',
+            connectionType: '',
+            place: '',
+            argument: null,
+            persistent: false,
+            protected: false,
+          },
+          registeredEndpoints: [],
+          Metro: null,
+          infoboxTitle: '',
+          infoboxMessage: '',
+        }
+      },
+      created () {
+        this.connectionTypeOptions = {
+          serial: 'Serial',
+          udpin: 'UDP Server',
+          udpout: 'UDP Client',
+          tcpin: 'TCP Server',
+          tcpout: 'TCP Client',
+        }
+      },
+      mounted () {
+        Metro.init()
+        this.Metro = Metro
+        this.fetchRegisteredEndpoints()
+        setInterval(this.fetchRegisteredEndpoints, 5000)
+      },
+      methods: {
+        parseEndpointFromAPI (endpoint) {
+          return {
+            name: endpoint.name,
+            owner: endpoint.owner,
+            connectionType: endpoint.connection_type,
+            place: endpoint.place,
+            argument: parseInt(endpoint.argument),
+            persistent: endpoint.persistent,
+            protected: endpoint.protected,
+          }
+        },
+        parseEndpointToAPI (endpoint) {
+          return {
+            'name': endpoint.name,
+            'owner': endpoint.owner,
+            'connection_type': endpoint.connectionType,
+            'place': endpoint.place,
+            'argument': endpoint.argument,
+            'persistent': endpoint.persistent,
+            'protected': endpoint.protected,
+          }
+        },
+        fetchRegisteredEndpoints () {
+          axios({
+            method: 'get',
+            url: '/v1.0/endpoints',
+            timeout: 500,
+          })
+            .then((response) => {
+              this.registeredEndpoints = response.data.map((endpoint) =>
+                this.parseEndpointFromAPI(endpoint)
+              )
+            })
+            .catch(() => {
+              this.infoboxTitle = 'Error'
+              this.infoboxMessage = 'Cannot fetch points. Check you network connection.'
+              Metro.infobox.open('#infobox')
+            })
+        },
+        createEndpoint () {
+          axios({
+            method: 'post',
+            url: '/v1.0/endpoints',
```

Comment:
We should have the root URL as a parameter to avoid the duplicated /version/ in all endpoints.